build.gradle

```diff
@@ -48,6 +48,13 @@ subprojects {
         useJUnitPlatform()
     }
 
+    java {
+        toolchain {
+            languageVersion = JavaLanguageVersion.of(11)
+            vendor = JvmVendorSpec.ADOPTOPENJDK
```

Comment:
Does this require that contributors use Adopt specifically? If so, why do that?

Reply:
Basically Gradle downloads an adopt open JDK 11 if no compatible JDK is installed.
Yeah we could drop the vendor. This toolchain feature sounds for me like a possibility to reduce "but it works on my machine" statements. 

https://docs.gradle.org/current/userguide/toolchains.html
https://blog.gradle.org/java-toolchains

Reply:
> This toolchain feature sounds for me like a possibility to reduce "but it works on my machine" statements.

If that's the problem you want to solve, why not introduce a very basic CI using Github Actions? You could use the Gradle workflow.

Reply:
It's just a thing I used elsewhere, too.
I just want to avoid pushing 10th or more times for a pr.
Image some dev want's to extend something on a piece of software. It builds on his machine but fails in pipeline. Figuring out what it is and trying to fix that is faster locally instead of waiting until the pipeline runs through.